src/css/app.css

```diff
@@ -295,6 +298,11 @@ label > * {
     align-items: center;
 }
 
+.sidebar #page-loader,
+.mobile #page-loader {
+    min-width: unset !important;
```

Comment:
Is it possible to avoid `!important` because it's harder to debug since it overrides everything even if it has a higher specificity.

Reply:
Yeah. I'm actually not sure why I added that. It must have not worked for me when I was originally testing, but it works now.